_opcodes/op003.html

```diff
@@ -10,10 +10,16 @@
 iwd2: 0
 pst: 1
 ---
-Applies the berserk effect to the targeted creature(s).
+Applies the berserk effect to the targeted creature(s) in the style specified by the 'Type' parameter.
+<br/>Known values for 'Type' are:
+<br/>0 Default/In Combat -> Only berserk while in combat.
+<br/>1 Constant -> always berserk, sets SPELLSTATE 39 (only in EEs).
+<br/>2 Blood Rage -> always berserk, sets SPELLSTATE 35 (only in EEs)
 <br/>Target will uncontrollably attack enemies if any are in view, and will occasionally
 attack neutral creatures.
 
+In Combat(0) = Only berserk while in combat. Constant(1) = always berserk, sets SPELLSTATE 39. Usable against party members and non-party members. Forces creature to use a melee weapon if possible.
```

Comment:
The first sentence is just a duplicate of the above list. Also, it's not clear if the second sentence is limited to certain modes or not.